back/engines/free/email_campaigns/app/mailers/email_campaigns/manual_campaign_mailer.rb

```diff
@@ -64,7 +64,7 @@ def home_url
     end
 
     def fix_image_widths(html)
-      doc = Nokogiri::HTML(html)
+      doc = Nokogiri::HTML.fragment(html)
```

Comment:
Without `.fragment`, Nokogiri will create a whole HTML page including the invalid `<!DOCTYPE html>`.